Automattic/wp-calypso
# Copy site: Update card for the copied site

Closes https://github.com/Automattic/dotcom-forge/issues/9578
Related to: https://github.com/Automattic/dotcom-forge/issues/9468

## Proposed Changes

This PR addresses the two following issues:

* Updates font type and font size on the `Staging site` card and `Copy site` card

| Before | After |
|--------|--------|
<img width="756" alt="Screenshot 2024-10-29 at 2 27 44 PM" src="https://github.com/user-attachments/assets/a8b5e63f-ff5b-44e1-98c7-864ea414982e"> | <img width="730" alt="Screenshot 2024-10-29 at 2 27 20 PM" src="https://github.com/user-attachments/assets/00a04ee3-e60a-40ea-8706-b9bb660d1dfe">

| Before | After |
|--------|--------|
<img width="693" alt="Screenshot 2024-10-29 at 2 28 39 PM" src="https://github.com/user-attachments/assets/d203a56e-cfc5-4feb-b448-2200caa53777"> | <img width="692" alt="Screenshot 2024-10-29 at 2 28 33 PM" src="https://github.com/user-attachments/assets/c8e79db8-41ec-4b3d-8847-9f8afb8781fd">

* Updates the graphic to be `WordPress blue` on the `Copy site` card

## Testing Instructions

**To test a staging site card**

* Ensure that you have a staging site
* Navigate to the Calypso Live link located below
* Navigate to `home/{yourStagingSite}`
* Observe the staging card on the top
* Confirm that you can see the correct font, size and color (Figma: NNfIg8OP1Xx0kumwkd2if8-fi-0_1 )

**To test a copy site celebration card**

* Ensure that you have a copy site (you can create one from `settings/general/{yourAtomicSite}` or `Settings > General > Copy site` on an Atomic site)
* * Navigate to the Calypso Live link located below
* Navigate to `home/{yourCopySite}`
* Observe the copy card on the top
* Confirm that you can see the correct font, size, color and graphic (Figma: NNfIg8OP1Xx0kumwkd2if8-fi-0_1 )

## Pre-merge Checklist
